src/web_interface/rest/rest_base.py

```diff
@@ -9,17 +9,19 @@
 from web_interface.rest.rest_compare import RestCompare
 from web_interface.rest.rest_file_object import RestFileObject
 from web_interface.rest.rest_firmware import RestFirmware
+from web_interface.rest.rest_missing_analyses import RestMissingAnalyses
 from web_interface.rest.rest_statistic import RestStatus
 
 
 class RestBase:
     def __init__(self, app=None, config=None):
         self.api = Api(app)
         self.api.add_resource(RestBinary, '/rest/binary/<uid>', methods=['GET'], resource_class_kwargs={'config': config})
+        self.api.add_resource(RestBinarySearch, '/rest/binary_search', '/rest/binary_search/<search_id>', methods=['GET', 'POST'], resource_class_kwargs={'config': config})
         self.api.add_resource(RestCompare, '/rest/compare', '/rest/compare/<compare_id>', methods=['GET', 'PUT'], resource_class_kwargs={'config': config})
-        self.api.add_resource(RestFirmware, '/rest/firmware', '/rest/firmware/<uid>', methods=['GET', 'PUT'], resource_class_kwargs={'config': config})
         self.api.add_resource(RestFileObject, '/rest/file_object', '/rest/file_object/<uid>', methods=['GET'], resource_class_kwargs={'config': config})
-        self.api.add_resource(RestBinarySearch, '/rest/binary_search', '/rest/binary_search/<search_id>', methods=['GET', 'POST'], resource_class_kwargs={'config': config})
+        self.api.add_resource(RestFirmware, '/rest/firmware', '/rest/firmware/<uid>', methods=['GET', 'PUT'], resource_class_kwargs={'config': config})
+        self.api.add_resource(RestMissingAnalyses, '/rest/missing_analyses', methods=['GET'], resource_class_kwargs={'config': config})
```

Comment:
Hm. Maybe `/rest/missing` suffices. But if you want to keep analysis in you have to make it singular to adhere to REST best practices (i.e. `/rest/missing_analysis`).

Reply:
changed to `/rest/missing`